### ISSUE TITLE
Inconsistent database engine result in failed FK creation

### ISSUE THREAD

**Issue by noogen** [cytech/BillingTrack#59]
Receiving the following error during initial migration using`php artisan migrate:fresh --seed` or visit `your.site.com/setup`

```
[Illuminate\Database\QueryException]
SQLSTATE[HY000]: General error: 1005 Can't create table xxx.#clients (errno: 150 "Foreign key constraint is incorrectly formed") (SQL: alter table `clients` add constraint `clients_industry_id_foreign` foreign key (`industry_id`) references `industries` (`id`) on delete no action on update no action)
```

**To Reproduce**
1. clone project
2. run through setup instruction `.env` file, database creation, etc....
3. finally run `php artisan migrate:fresh --seed` or visit `your.site.com/setup`

**Expected behavior**
Should not error.

**Desktop/Smartphone:**
Should not matter.

**Additional context**
Issue is due to database engine difference between `clients` [InnoDB](https://github.com/cytech/BillingTrack/blob/master/database/migrations/2018_08_17_000013_create_clients_table.php#L25) and `industries` [MyISAM](https://github.com/cytech/BillingTrack/blob/master/database/migrations/2019_05_29_000100_version_510.php#L26) table. 

Instead of manually define database engine everywhere, I recommend to define it at the global/configuration area?  https://github.com/cytech/BillingTrack/blob/master/config/database.php#L25

```php
        'mysql' => [
            'driver' => 'mysql',
            'host' => env('DB_HOST'),
            'port' => env('DB_PORT', '3306'),
            'database' => env('DB_DATABASE'),
            'username' => env('DB_USERNAME'),
            'password' => env('DB_PASSWORD'),
            'unix_socket' => env('DB_SOCKET', ''),
            'charset' => 'utf8mb4',
            'collation' => 'utf8mb4_unicode_ci',
            'prefix' => '',
            'strict' => env('DB_STRICT', false),
            'engine' => 'innodb',
            //'charset'   => 'utf8',
            //'collation' => 'utf8_unicode_ci',

        ],
```

**Comment by cytech** [comment]
interesting. wonder why this has never popped up before. Probably my Mysql server defaults to the default innodb.
i assume your default mysql server engine is set to MyISAM ? so the migrations that do not specify ->engine are going myISAM ?
anyhow I agree on the above change.
thank you

**Comment by noogen** [comment]
Yes, I created my database from the `cpanel` wizard and it defaulted to MyISAM.
